inference-engine/thirdparty/clDNN/src/primitive_inst.cpp

```diff
@@ -258,10 +258,16 @@ void primitive_inst::allocate_internal_buffers(void) {
             _intermediates_memory.push_back(engine.allocate_memory(layout, allocation_type::usm_host));
     }
 }
+memory::ptr primitive_inst::allocate_output(engine& _engine, std::unique_ptr<memory_pool>& pool, const program_node& _node,
+        bool is_internal, bool is_dummy) {
```

Comment:
I don't see how this `is_dummy` parameter is used. Do I miss something?

Reply:
Removed